tests/test_policies.py

```diff
@@ -72,6 +72,8 @@ def test_get_policy_and_config_classes(policy_name: str):
         ),
         # Note: these parameters also need custom logic in the test function for overriding the Hydra config.
         ("pusht", "act", ["env.task=PushT-v0", "dataset_repo_id=lerobot/pusht"]),
+        ("dora_aloha_real", "act_real", []),
```

Comment:
Hmm, okay I'm all for "the more the merrier" with testing. But the question now arises: why not test all possible configuration combinations. Can we add a note on this test for future contributors explaining what our strategy is (so they know whether to add tests or not)? Currently, it looks like it's: _"Here we test some parameter configurations for various combinations of policy and dataset. The combinations are by no means exhaustive, and for now we add tests as we see fit."_